openlibrary/plugins/upstream/utils.py

```diff
@@ -709,51 +709,63 @@ def strip_accents(s: str) -> str:
 
 
 @functools.cache
-def get_languages() -> dict:
-    keys = web.ctx.site.things({"type": "/type/language", "limit": 1000})
+def get_languages(limit: int = 1000) -> dict:
```

Comment:
Is a limit greater than 1,000 really needed for this dataset?
* https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

Reply:
Probably not, doesn't change functionality either way so up to approvers whether or not it's kept. Just threw it in as a bit of a clarification of what's happening within the function.